OracleDatabase/RAC/OracleConnectionManager/dockerfiles/21.3.0/client.rsp

```diff
@@ -0,0 +1,106 @@
+###############################################################################
+## Copyright(c) Oracle Corporation 1998,2018. All rights reserved.           ##
```

Comment:
Please change the copyright years to be accurate. 1998 is not the year this file was created and it was updated in 2022. It should also refer to "Oracle and/or its affiliates" and please remove "All rights reserved".